default.nix

```diff
@@ -1,35 +1,60 @@
-{ rocksDBVersion ? "6.29.3" }:
```

Comment:
Can we please add two tasks to JIRA:
1. Check forward and backward compatibility between nix `aleph-node` and current `aleph-node` 
2. Make sure WASM produced by nix `aleph-node` is exactly the same as WASM produced by current node.

Reply:
Added.